### PR TITLE
network tests: test MTU on the secondary interface with no dhcp server

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:

This PR is adding a test to vmi_multus_tests to make sure the correct MTU is propagated to the vm interface in case there is no dhcp server running on the virt-launcher pod.

The test is especially important to verify that libvirt upgrade in PR https://github.com/kubevirt/kubevirt/pull/5147, which includes MTU bug fix, preserved the functionality of passing the MTU of unmanaged tap device to qemu. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/network/vmi_multus.go [hunk 1]
```diff
@@ -466,6 +466,32 @@ var _ = SIGDescribe("[Serial]Multus", func() {
...
+
+				vmi := libvmi.NewFedora(
+					libvmi.WithInterface(linuxBridgeInterface),
+					libvmi.WithNetwork(linuxBridgeNetwork),
+				)
+
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(runSafeCommand(vmiOne, fmt.Sprintf("cat %s | grep %s\n", mtuPath, getPodInterfaceMtu(vmi)))).To(Succeed())
```
Comment (qinqon) [inline tests/network/vmi_multus.go:493]: Retrieve Pod MTU and vmi MTU and use an Equal gomega matcher so it's clear what is failing, failing Greg is not that bad but not as good as an explicit matcher.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: tests/network/vmi_multus.go (hunk 1 above)
Comment (AlonaKaplan) [reply tests/network/vmi_multus.go]: done

**Review by enp0s3 — APPROVED** [review]
/approve

**Review by EdDev — COMMENTED** [review]
Looks good!
I only have one comment here: We do not seem to enforce the mtu on the secondary interface, therefore, if the default one is used (1500), the test may pass just because the guest will probably use the same default and not because the mtu was propagated though libvirt into the guest.

Holding this until you can confirm this point. Feel free to unhold when you are ready.
/hold

**Comment by AlonaKaplan** [comment]
> Looks good!
> I only have one comment here: We do not seem to enforce the mtu on the secondary interface, therefore, if the default one is used (1500), the test may pass just because the guest will probably use the same default and not because the mtu was propagated though libvirt into the guest.
> 
> Holding this until you can confirm this point. Feel free to unhold when you are ready.
> /hold

You're totally correct. Added MTU to the nad.

/hold cancel

**Review by EdDev — APPROVED** [review]

**Pull request merged**